tests/screenshot/apps/components/FlexiblePopupPanels.js

```diff
@@ -79,6 +79,10 @@ const EachPanel = withConfig(
 			title: 'with transparent scrim',
 			component: <FlexPopupPanels open scrimType="transparent" />
 		},
+		{
+			title: 'with translucent scrim',
+			component: <FlexPopupPanels open scrimType="translucent" />
+		},
```

Comment:
Default value of scrimType is translucent.
This is already being tested in the code 74~77 lines.
Please remove this test.

Reply:
Done.